pdr_backend/pdr_dashboard/callbacks/callbacks_feeds.py

```diff
@@ -207,8 +202,16 @@ def update_graphs(is_open, selected_rows, feeds_table_data):
         feed_figures: FeedModalFigures = get_feed_figures(payouts, subscriptions)
 
         children = [
-            feeds_page.get_feed_graphs_modal_header(selected_row),
-            feeds_page.get_feed_graphs_modal_body(feed_figures.get_figures()),
+            get_graphs_modal_header(
```

Comment:
ok, then to simplify, I think you can use just "feeds-modal"/"predictoors-modal" and selected row as the parameter to get_graphs_modal_header and get_graphs_modal_body. Assemble based on case, inside the function itself, to keep more code in helpers than in callbacks files.